src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partnerStatus = `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`;
```

Comment:
Look at the checklist point `1. [CODE STYLE] - Use destructuring for getting access to values of props object`